### ISSUE TITLE
PT BG 2016

### ISSUE THREAD

**Issue by RosaLeeVV** [Open-Family-Policy-Platform/EU-SILC_CS#46]
The values of 15 days and 90% of earnings are the same in 2016 as in 2018. 
However, minimum: €235.16 for the duration of PT and ceiling: €1,329.18 for the duration of PT is not found for 2016. Therefore, I believe these values may need to be removed but I am not sure if I should remove them or if you would prefer to do so. Please let me know if I should just be deleting these items as I go.

**Comment by alzbetab** [comment]
Yes, you can delete them. I will still see the changes you make and can revert them if there is a problem.
